apps/common-app/src/examples/InvalidValueAccessExample.tsx

```diff
@@ -9,8 +9,7 @@ import Animated, {
 } from 'react-native-reanimated';
 
 configureReanimatedLogger({
-  // change to `false` or remove the `configureReanimatedLogger` call to
-  // disable the warning
+  // change to `false` to disable the warning
   strict: true,
```

Comment:
I decided to make logger `strict` by default. I think that nobody will enable strict mode as nobody reads docs until something doesn't work, thus no one will see my new invalid value read/write warning from the #6310 PR.